src/emit/expression.rs

```diff
@@ -2120,3 +2122,227 @@ fn runtime_cast<'a>(
         _ => unreachable!(),
     }
 }
+
+/// Emit expression into a slice
```

Comment:
Nitpick: Superfluous. A more useful comment could for example say when this is needed or explain the return type (which I'd guess is a pointer to the slice and the length of the slice)?

Reply:
You're right, that comment is terrible.